apps/k8s-io/configmap-nginx.yaml

```diff
@@ -269,7 +269,7 @@ data:
           rewrite ^/needs-ok-to-test$ https://github.com/pulls?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Apr+label%3Aneeds-ok-to-test+label%3A%22cncf-cla%3A+yes%22+-label%3Aneeds-rebase redirect;
           rewrite ^/oncall$          https://storage.googleapis.com/test-infra-oncall/oncall.html redirect;
           rewrite ^/oncall-hotlist$  https://github.com/kubernetes/test-infra/search?q=label%3Akind%2Foncall-hotlist+is%3Aopen&type=Issues redirect;
-          rewrite ^/owners$          https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md redirect;
+          rewrite ^/owners$          https://www.kubernetes.dev/docs/guide/owners/ redirect;
```

Comment:
@sftim it is kubernetes.dev. k8s.dev is a redirect to kubernetes.dev and we redirect straight to kubernetes.dev